### ISSUE TITLE
Sequences with different lengths

### ISSUE THREAD

**Issue by asierFernandezP** [tseemann/snp-dists#52]
Hi,

Is it possible to perform the snp-dists analysis with a fasta file of sequences with different lengths as input?

Thanks in advance,

Asier

**Comment by kloetzl** [comment]
No. snp-dists requires all sequences to be of identical length. If your sequences are of varying length you have to align them first.
